### DISCUSSION TITLE
ARM Architheture

### DISCUSSION THREAD

**Discussion by brunomarram** [martin-helmich/prometheus-nginxlog-exporter#250] [Ideas]
Hello Everyone.

I need this exporter to create one grafana dashboard for my company some days ago and i need that for one ARM machine. For that, i just clone the repo in it and build the executable in there.

It will be interesting that this package stay disponible at releases page for avoid this type of work.

If its necessary, i can send the binary file that i was generated. Regards.

**Comment by martin-helmich** [comment]
Thanks for the suggestion! :+1:  I've created https://github.com/martin-helmich/prometheus-nginxlog-exporter/issues/253 to track this request. Since this project already uses Goreleaser, which supports multi-arch builds anyways, this shouldn't be too much of a change.

**Comment by vterdunov** [comment]
#253 closed. @martin-helmich Can you make new relese with arm support?
